client/app/controllers/index.js

```diff
@@ -16,22 +16,12 @@ export default Ember.Controller.extend({
             }));
         },
         addCollection () {
-
-            let coll = {
-                title: this.get('newCollectionTitle'),
-                tags: '',
-                description: '',
-                createdBy: 'Marcia Gonzales',
-                dateCreated:  new Date(),
-                dateUpdated: new Date(),
-                items: []
-            };
-            $.post('/api/collections', coll).then(result => {
```

Comment:
Oh, I accidentally overlooked this redirect so right now it will stay on the collections page and add the new collection to the bottom of the list